app/assets/stylesheets/global.css.scss

```diff
@@ -0,0 +1,125 @@
+$dark_gray: #2B2E2E;
+$light_gray: #C0C1C1;
+$green: #33CD99;
+
+@import url(http://fonts.googleapis.com/css?family=Amaranth:700italic);
+
+@mixin bold_font {
+	font-family: "Brandon Grotesque","Helvetica Neue", Helvetica, sans-serif;
+	font-weight: bold;
+	text-transform: uppercase;
+}
+
+* {
+	box-sizing: border-box;
+	-webkit-font-smoothing: antialiased;
+}
+
+body {
+	padding-top: 0rem;
+	font-family: "Open Sans", "Helvetica Neue", Helvetica, sans-serif;
+}
+
+.wrapper {
+	width: 80%;
+	margin: 0 auto;
+}
+
+.wrapper_with_padding {
+	width: 80%;
+	margin: 0 auto;
+	padding: 3.5rem 0;
+}
+
+.clearfix:before, .clearfix:after {
+	content: " ";
+	display: table;
+}
+
+.clearfix:after {
+	clear: both;
+}
+
+button, .button {
+	border: none;
+	outline: none;
+	background: $green;
+	padding: 1rem 3.5rem;
+	border-radius: 2rem;
+	a {
+		color: white;
+		text-decoration: none;
+	}
+}
+
+.button {
+	color: white;
+	margin: 2rem 0;
+}
+
+a {
+	color: $green;
+	text-decoration: none;
+	&:hover {
+		color: $dark_gray;
+	}
+}
+
+header {
+	background: $dark_gray;
+	position: fixed;
+	top: 0;
+	width: 100%;
+	padding: 1.75rem 0;
+	border-bottom: 7px solid $green;
+	.header_inner {
+		margin: 0 auto;
+		width: 90%;
+		#logo {
+			display: inline-block;
+			float: left;
+			width: 10rem;
+		}
+		nav {
+			float: right;
+			a {
+				color: white;
+				font-size: 1.1rem;
+				line-height: 1.5;
+				margin-left: 1rem;
+				text-decoration: none;
+				&:hover {
+					color: $green;
+				}
+			}
+		}
+	}
+}
+
+footer {
+	background: $dark_gray;
+	width: 100%;
+	padding: 2rem 0;
+	color: white;
+	text-align: center;
+	@include bold_font;
+	p {
+		font-size: 0.85rem;
+		margin: 0;
+	}
+}
+
+input[type="text"], input[type="email"], input[type="password"], textarea {
+	width: 100%;
+	margin-bottom: 1rem;
+	border: 1px solid $light_gray;
```

Comment:
Line should be indented with spaces, not tabs<br>Name of variable `light_gray` should be written in all lowercase letters with hyphens instead of underscores